exec_test.go

```diff
@@ -194,7 +194,7 @@ func TestCommandStartWait(t *testing.T) {
 	testCases := map[string]testCase{
 		// Background context
 		"success":                     {distro: &realDistro, cmd: "exit 0"},
-		"failure fake distro":         {distro: &fakeDistro, cmd: "exit 0", wantErrOn: AfterWait},
```

Comment:
TODO (in another PR maybe), subtests can be read as sentences when being more than one word or so, like "error when blablablabla". I find it easier to read.

Not related to that PR, but anther idea for code garderning!

Reply:
🌱🪴🌲